checkov/terraform/checks/provider/panos/credentials.py

```diff
@@ -17,6 +17,8 @@ def __init__(self) -> None:
     def scan_provider_conf(self, conf: Dict[str, List[Any]]) -> CheckResult:
         if self.secret_found(conf, "api_key", panos_api_key_pattern):
             return CheckResult.FAILED
+        if self.secret_found(conf, "password", panos_password_pattern):
```

Comment:
no need for the pattern check, you can just check, if something is written for the `password` value
```suggestion
        if conf.get("password"):
```
and please add a test to the test file 🙂